src/test/osd/osd-fast-mark-down.sh

```diff
@@ -99,12 +98,10 @@ function test_fast_kill() {
      if [ $down_osds -lt $i ]; then
         echo Killed the OSD, yet it is not marked down
         ceph osd tree
-        teardown $dir 
         return 1
      fi
    done
    pkill -SIGTERM rados
-   teardown $dir || return 1
```

Comment:
I agree with @branch-predictor . This is not typical of the structure of other tests but that's ok.